MovieApp/MovieCollectionView.swift

```diff
@@ -55,13 +65,57 @@ class MovieCollectionView: UIView {
             make.top.leading.trailing.bottom.equalToSuperview()
         }
         collectionView.collectionViewLayout.invalidateLayout()
+        
+        collectionView.contentInset = UIEdgeInsets(top: 0, left: 20, bottom: 0, right: 0)
+    }
+
+    func populateMovieData() {
```

Comment:
I wouldn't do it like this. Because this view should not call the network call. It should receive the list of movies to show. This logic belongs in the view controller. View controller should decide what type of movie group to fetch and to show in which collection view.